Makefile

```diff
@@ -37,15 +44,20 @@ embedmd:
 docs: embedmd
 	embedmd -w `find Documentation -name "*.md"`
 
-assets: gobindata
```

Comment:
assets was actually a phony target that duplicated the name of an real directory in the repo. this was confusing